var/spack/repos/builtin/packages/adios2/package.py

```diff
@@ -119,10 +119,20 @@ class Adios2(CMakePackage):
     # See https://github.com/ornladios/ADIOS2/pull/1899
     patch('2.5-fix-clear_cache.patch', when='@2.5.0')
 
+    @when('%fj')
+    def patch(self):
+        """ add fujitsu mpi commands #16864 """
+        f = join_path('cmake', 'upstream', 'FindMPI.cmake')
```

Comment:
I think this might be well placed in a CMake Upstream PR here:
- https://gitlab.kitware.com/cmake/cmake/
and then as a patch in the Spack `cmake` package?

Reply:
Update: oh I see, you patch the internally shipped copy of this file for older CMakes.

Reply:
Yes.
Currently, spack installed cmake is 3.17.1 or older.
So adios2 is used the internally shipped copy.